src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -134,8 +134,8 @@ protected HttpUrl getDownloadUrl() throws BlackDuckIntegrationException {
         StringBuilder url = new StringBuilder(blackDuckServerUrl.string());
         if (!blackDuckServerUrl.string().endsWith("/")) {
             url.append("/");
-            url.append(LATEST_SCAN_CLI_TOOL_DOWNLOAD_URL);
         }
+        url.append(LATEST_SCAN_CLI_TOOL_DOWNLOAD_URL);
```

Comment:
Fixes IDETECT-4461, which was a bug that fails to correctly form the URL for the scan-cli download request when the blackduck.url provided has a trailing / character.